api/src/pcapi/alembic/versions/20230803T131455_ecc1eaadf729_remove_ff_wip_enable_collective_dms_tracking.py

```diff
@@ -0,0 +1,36 @@
+"""
+remove_ff_wip_enable_collective_dms_tracking
```

Comment:
Ce commentaire s'affiche dans la sortie de la commande `alembic history`. Il doit donc être lisible.

```suggestion
"""Remove WIP_ENABLE_COLLECTIVE_DMS_TRACKING feature flag
```

Reply:
Ok merci bien, je modifie 👍 !